editors/nvi/Portfile

```diff
@@ -2,7 +2,7 @@ PortSystem      1.0
 
 name            nvi
 version         1.81.6
-revision        6
+revision        7
```

Comment:
Are you fixing only build errors or also runtime errors? The only Trac ticket you referenced was a build error. If you are only fixing build errors, then the revision should not be increased. (See the documentation for the revision keyword in the MacPorts guide.)

Reply:
I have removed this change in my latest commit since I was only trying to fix build errors.